editor/js/reaction.js

```diff
@@ -679,7 +825,12 @@ function initValidateHandlers() {
   });
 }
 
-// Convert a Message_Field name from a data-proto attribute into a proto class.
+/**
+ * Converts a Message_Field name from a data-proto attribute into a proto class.
+ * @param {string} protoName Underscore-delimited protocol buffer field name,
+ *     such as Reaction_provenance.
+ * @return {!typeof jspb.Message|null}
```

Comment:
Is this the correct way to record the type, or should it be `{?typeof jspb.Message}`?

Reply:
Done.